submissions/NwagboJohn/React_task/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Also submit React task in a separate PR. You don't want this to block your PHP task from getting merged